src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/tools/SetClrPath.cmd

```diff
@@ -1,6 +1 @@
 set clrpath=%MANAGED_TOOLS_ROOT%\%MANAGED_REFS_VERSION%
```

Comment:
I didn't know how to get clrpath automatically. It's the path used by sn.exe but I couldn't find where to get it.